### PR TITLE
Merge 3.3 into 4.0 for the final time

### PR DESCRIPTION
**Pull request by dasgarner**
Henceforth we should either make modifications in both branches or cherry pick when feasible.

### CODE REVIEW

**Comment by dasgarner** [comment]
I've checked through all of the changes made to files in v3 which have been deleted in v4, and I think they are all ported across correctly.

**Review by PeterMis — COMMENTED** [review]
Looks sensible to me I think, just a few comments

File: views/alphavantage-form-settings.twig [hunk 1]
```diff
@@ -37,6 +37,10 @@
         {{ forms.checkbox("isPaidPlan", title, interface.getSetting("isPaidPlan"), helpText) }}
     {% endif %}
 
+    {% set title %}{% trans "Paid plan?" %}{% endset %}
```
[the remark this answers is not in the document]
Comment (dasgarner) [reply views/alphavantage-form-settings.twig:40]: Ah interesting, I must have missed that one. I'll resolve.

File: views/user-welcome.twig [hunk 2]
```diff
@@ -195,53 +202,76 @@
                     template: no_nav_template
                 },
                 { // 18
+                    content: "{% trans "Click on the Add button to create one Region!" %}",
```
[the remark this answers is not in the document]
Comment (PeterMis) [reply views/user-welcome.twig:205]: That sounds like a good plan to me, I wasn't sure if the user-welcome was changed in v4 or not :)

**Review by PeterMis — APPROVED** [review]

**Pull request merged**